app/controllers/auth.php

```diff
@@ -70,6 +70,10 @@ public function login($return = '')
             }
         }
 
+       if(!empty(conf('manage_ipwhitelist'))) { 
+         check_ip($_SERVER['REMOTE_ADDR']); 
```

Comment:
You need to use `getRemoteAddress()` instead of `$_SERVER['REMOTE_ADDR']` to fix requests that come from a loadbalancer/proxy